Online-Messaging-App/src/app/home/chatbox/channel-user-list/channel-user-list.component.html

```diff
@@ -1,40 +1,44 @@
 <mat-card class="scrollMe">
```

Comment:
Also this list scrolls, even when not long enough to. can you fix please

Reply:
Just put comment at top of the class cause I don't know where the change would be